simplepath/utils.py

```diff
@@ -0,0 +1,35 @@
+"""Module that describes all utility functions for Simplepath."""
+
+
+def convert_object_to_dict(data_object):
+    """Convert all of the non-function attributes of an object into
+       a dictionary.
+    Args:
+        data_object(object): any Python object
+    Returns:
+        The dictionary representation of all fields in the Python object.
+    """
+
+    if hasattr(data_object, "__dict__"):
+        object_dict = vars(data_object).copy()
+        for attrib_name, attrib_val in object_dict.items():
+            if isinstance(attrib_val, tuple):
+                object_dict[attrib_name] = [convert_object_to_dict(element)
+                                            for element in attrib_val]
+                object_dict[attrib_name] = tuple(object_dict[attrib_name])
```

Comment:
this is not necessary since json (which simplejson maps) does not have a concept for a tuple....